helm-configs/glance/glance-helm-overrides.yaml

```diff
@@ -977,7 +977,7 @@ manifests:
   pod_rally_test: false
   pvc_images: true
   network_policy: false
-  secret_db: true
+  secret_db: false
```

Comment:
If this is false, how is db init happening ? will the init job not need the right secret for glance-admin to be present ?

Reply:
So for historical context I've written up some info in the commit. https://github.com/rackerlabs/genestack/pull/101/commits/11fa3627344179a821967040ed4e3494be2a74bf

If you look farther up in this config file you'll see `job_db_init: false` which is actually set in every single chart you guys use today. The reason is you're using the MariaDB operator for those operations.

Reply:
So, I see in our cluster secret nova-db-admin being created. This is created by nova/templates/secret-db.yaml which is used by the db_sync. So I am still not sure if secres_db false will not cause problem. Is this something you have tested ?